main/inc/lib/main_api.lib.php

```diff
@@ -43,6 +43,7 @@
 define('COURSE_STUDENT', 14);   //student subscribed in a course
 define('SESSION_STUDENT', 15);  //student subscribed in a session course
 define('COURSE_TUTOR', 16); // student is tutor of a course (NOT in session)
+define('ROLE_INVITED', 20);
```

Comment:
ROLE_INVITED? What does it mean? Use a more descriptive name. 
Don't need to add the "ROLE_" prefix + add comments.